crates/sui-rpc-loadgen/src/main.rs

```diff
@@ -193,5 +196,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
     };
     load_test.run().await?;
 
+    load_test.processor.dump_cache_to_file();
```

Comment:
nit: we should minimize the logic in `main` file, can we put it in `load_test.rs`?

Reply:
makes sense, will do